enumerations.xml Amend the CS of the code examples

mmalferov opened the pull request.

mmalferov committed:
enumerations.xml Amend the CS of the code examples

tiffany-taylor commented on language/enumerations.xml:
```diff
@@ -952,8 +980,13 @@ enum UserStatus: string
 
 foreach (UserStatus::cases() as $case) {
-    printf('<option value="%s">%s</option>\n', $case->value, $case->label());
+    printf(
+        "<option value=\"%s\">%s</option>\n",
+        $case->value,
+        $case->label()
+    );
```
I'm not sure I agree with breaking this out into separate lines. If I recall correctly, this isn't done elsewhere in the manual in similar situations, but do correct me if I'm wrong.

mmalferov replied:
Rolled back

tiffany-taylor commented:
Changes in this PR are made to the pages in this section: https://www.php.net/manual/en/language.enumerations.php

mmalferov committed:
Update enumerations.xml

I was guided by the reader and how convenient it would be for the user to read the example. I'm going back to the previous formatting, since it violates the tradition :)

vrana commented:
I see no point in adding newlines before `?>`.

Crell commented:
Please remove the newlines before `?>`, as we don't need those, and rebase the PR.  We should be able to merge it then.  Thanks.

mmalferov committed:
Update enumerations.xml Remove the newlines before ?>

mmalferov committed:
Merge branch 'master' into patch-996250

mmalferov committed:
Merge branch 'master' into patch-996250
